src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/OutgoingTransfersControllerTests.cs

```diff
@@ -628,14 +665,37 @@ public async Task EditTonnagesGet_GivenMappedModel_ModelShouldBeReturned()
         public async Task EditTonnagesGet_ShouldReturnView()
         {
             //act
+            var evidenceNoteId = TestFixture.Create<Guid>();
+            var note = TestFixture.Create<TransferEvidenceNoteData>();
+            note.Status = NoteStatus.Draft;
+            A.CallTo(() => weeeClient.SendAsync(A<string>._,
+                    A<GetTransferEvidenceNoteForSchemeRequest>.That.Matches(r => r.EvidenceNoteId == evidenceNoteId))).Returns(note);
             var result =
-                await outgoingTransferEvidenceController.EditTonnages(organisationId, TestFixture.Create<Guid>(), TestFixture.Create<bool?>()) as
+                await outgoingTransferEvidenceController.EditTonnages(organisationId, evidenceNoteId, TestFixture.Create<bool?>()) as
                     ViewResult;
 
             //assert
             result.ViewName.Should().Be("EditTonnages");
         }
 
+        [Fact]
+        public async Task EditTonnagesGet_ShouldReturnManageEvidenceView()
+        {
+            //act
+            var evidenceNoteId = TestFixture.Create<Guid>();
+            var note = TestFixture.Create<TransferEvidenceNoteData>();
+            note.Status = NoteStatus.Submitted;
+            A.CallTo(() => weeeClient.SendAsync(A<string>._,
```

Comment:
Can do a theory
[Theory]
[ClassData(typeof(NoteStatusCoreData))] and pass Core.AatfEvidence.NoteStatus status as test parameter.

And just do if something like in the test to ignore the two status

if (status.Equals(Core.AatfEvidence.NoteStatus.Returned) ||
                status.Equals(Core.AatfEvidence.NoteStatus.Draft))
            {
                return;
            }